docs/source/tutorials.rst

```diff
@@ -168,90 +168,90 @@ You may want to define your own survey if you wish to modify some parameters or
   }
 
   Rubin = btk.survey.Survey(
-      "LSST",
+      "Rubin",
```

Comment:
I do not understand this change. The observatory is called Vera C. Rubin but the survey is still called LSST.

Reply:
That is a good point! I'll revert it, thanks @aboucaud :)